sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/OfferRequirement.java

```diff
@@ -59,42 +62,39 @@ public static OfferRequirement create(
             Collection<TaskRequirement> taskRequirements,
             ExecutorRequirement executorRequirement,
             Optional<PlacementRule> placementRuleOptional) {
-
-        return new OfferRequirement(taskType, index, taskRequirements, Optional.of(executorRequirement),
-                placementRuleOptional);
+        return new OfferRequirement(
+                taskType, index, taskRequirements, Optional.of(executorRequirement), placementRuleOptional);
     }
 
     /**
-     * Creates a new {@link OfferRequirement} with provided executor requirement and empty placement
-     * constraints.
+     * Creates a new {@link OfferRequirement} with provided executor requirement and empty placement constraints.
      *
      * @see #OfferRequirement(String, Integer, Collection, Optional, Optional)
      */
     public static OfferRequirement create(
             String taskType,
-            int index,
+            Integer index,
```

Comment:
That's from master, it looks like? Not sure what's going on here. I can change it to `int`, though.